src/main.cpp

```diff
@@ -1738,6 +1816,22 @@ bool AcceptableInputs(CTxMemPool& pool, CValidationState& state, const CTransact
     return true;
 }
 
+bool GetOutput(const uint256& hash, unsigned int index, CValidationState& state, CTxOut& out)
+{
+    libzerocoin::ZerocoinParams* params = Params().Zerocoin_Params(false);
+    PublicCoinSpend ret(params);
```

Comment:
Are `params` and `ret` supposed to be doing something that the implementation has forgotten about? Or maybe leftovers from old code?

Reply:
just leftovers 👍 , the code was fetching + parsing the spend there and that was moved to zpivmodule.